langchain/agents/load_tools.py

```diff
@@ -262,9 +262,12 @@ def _get_scenexplain(**kwargs: Any) -> BaseTool:
     "google-serper": (_get_google_serper, ["serper_api_key", "aiosession"]),
     "google-serper-results-json": (
         _get_google_serper_results_json,
-        ["serper_api_key", "aiosession"],
+        ["serper_api_key", "serpapi_api_base_url", "aiosession"],
```

Comment:
>> Would it make sense to call it `serper_base_url` here?

yes, make sence.

>> Also it would require a change to serper.py, right?

yes, if we change google-serper, we need to change it.